workflow-templates/pr-name-check.yml

```diff
@@ -0,0 +1,31 @@
+name: 'PR name check'
+on:
+  pull_request:
+    types:
+      # Check title when opened.
+      - opened
+      # Check title when new commits are pushed.
+      # Required to use as a status check.
+      - synchronize
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: dylanvann/check-pull-request-title@v1
+        with:
+          # Match pull request titles in the form of:
+          # Release/s123 
+          # dev => canary
```

Comment:
dev => canary - is this one a thing? I don't think your regex actually supports it

Reply:
The comment just needs to be updated. This should not be a thing ideally. All `=>` PRs should go to dev as reflected in the regex

Reply:
Yeah the comment is wrong, removing it